assets/js/script.js

```diff
@@ -4,124 +4,80 @@ $(document).ready(function() {
   }).then(function(html) {
     html = html.replace(/assets\/img\//g, (window.baseUrl || '') + 'assets/img/')
     $('#custombar').before(html).remove()
-    activate()
+    render()
   })
 })
 
-function activate() {
-  var el = {
-    input: $('#letters-input'),
-    letterTabs: $('.tab-letter-color-letter'),
-    letterTab1: $('#tab-letter1'),
-    letterTab2: $('#tab-letter2'),
-    lettersWrapper: $('.tab-letter-wrapper.active'),
-    swatch: $('.swatch-image-border'),
-  }
-
-  Object.keys(el).forEach(function(key) {
-    if (!el[key].length) { console.warn(`No element present for ${key}`); }
-  })
-
-  var bagContainerHeight = $(".main-bag-container").width() * .7
-  $(".main-bag-container").height(bagContainerHeight)
-  var bagContainerHeight = $(".main-letters-container").width() * .7
-  $(".main-letters-container").height(bagContainerHeight)
-  var letterTabHeight = $(".swatch-row").height() / 2
-  $(".custom-letter-container").height(letterTabHeight)
+var state = {
```

Comment:
so this is mimicking the `state` and `setState` constructs of react. 

the idea is that we will keep all of the UI's state in a single object. then when we render, we'll just translate the state object into DOM.